package.json

```diff
@@ -4,10 +4,22 @@
   "dependencies": {
     "@frctl/fractal": "^1.1.1",
     "autoprefixer": "^6.7.2",
+    "eslint-config-idiomatic": "^3.1.0",
     "gulp": "^3.9.1",
+    "gulp-eslint": "~3.0.1",
+    "gulp-if": "~2.0.2",
     "gulp-plumber": "^1.1.0",
     "gulp-postcss": "^6.3.0",
     "gulp-sass": "^3.1.0",
-    "gulp-sourcemaps": "^2.4.1"
+    "gulp-sass-lint": "~1.3.2",
+    "gulp-sourcemaps": "^2.4.1",
+    "gulp-util": "~3.0.8",
+    "postcss-reporter": "~3.0.0",
+    "run-sequence": "~1.2.2",
+    "stylelint": "~7.10.1",
+    "stylelint-config-standard": "~16.0.0"
+  },
+  "scripts": {
+    "test": "./node_modules/eslint/bin/eslint.js *.js helper/*.js task/*.js"
```

Comment:
You mean to delete this `test` script and delete from `travis.yml` file line with `yarn test` or left only `yarn`?